.github/workflows/ci.yml

```diff
@@ -0,0 +1,175 @@
+name: EFCore.PG CI
+
+on: push
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest, windows-latest ]
+        dotnet: [ '3.0.100' ]
+        pg_major: [ 11, 10 ]
+        include:
+          - pg_major: 11
+            edb_version: '11.4-2'
+            postgis_version: '2.5.3'
+          - pg_major: 10
+            edb_version: '10.9-2'
+            postgis_version: '2.5.3'
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Start PostgreSQL ${{ matrix.pg_major }} (docker)
+        if: startsWith(matrix.os, 'ubuntu')
+        env:
+          IMAGE: npgsql/postgres:${{ matrix.pg_major }}
```

Comment:
Should this just be inlined into the `run`line below? Or is the environment variable actually needed somewhere?

Reply:
Setting the whole image in an environment variable is leftover from experimenting with a Windows image, so this could be inlined now. The matrix variables may still need transferred to an environment variable, but it'd be nice if not.

Reply:
Awesome, looks like matrix variables do resolve inline.